bootstrap/bootstrap.go

```diff
@@ -195,28 +198,64 @@ func (b *Bootstrap) applyEnvironmentChanges(environ *env.Environment, dir string
 	}
 }
 
-// Returns the absolute path to a global hook
-func (b *Bootstrap) globalHookPath(name string) string {
-	return filepath.Join(b.HooksPath, normalizeScriptFileName(name))
+// Returns the absolute path to the best matching hook file in a path, or os.ErrNotExist if none is found
+func (b *Bootstrap) findHookFile(hookDir string, name string) (string, error) {
+	if runtime.GOOS == "windows" {
+		// check for windows types first
+		if p, err := shell.LookPath(name, hookDir, ".BAT;.CMD;.EXE"); err == nil {
```

Comment:
Unfortunately we can't support powershell hooks yet, as need to write a environment hook magic thing for them.